lib/hedwig_mopidy/responders/mopidy.ex

```diff
@@ -69,12 +69,14 @@ defmodule HedwigMopidy.Responders.Mopidy do
   @usage """
 
 `dj playlists` lists all the available playlists
-`dj start <uri>` shuffles the playlist specified by <uri> (defaults to the Lab Zero playlist)
+`dj start` shuffles the last-played playlist, or Lab Zero default playlist
+`dj start <uri>` shuffles a publicly-available playlist
 `dj pause|stop` ceases playback
 `dj play|resume` starts playback
-`dj (what|who) ('s| is) playing` displays the currently playing track and playlist (e.g. what's playing or who is playing)
-`dj +1|:thumbsup:|:thumbsup_all:|:metal:|:shaka:|up|yes` upvotes if you like the currently playing track on the currently playing playlist
-`dj -1|:thumbsdown:|:thumbsdown_all:|down|no|skip|next` votes against the currently playing track on the currently playing playlist and skips to the next track
+`dj who's playing` or what's playing, shows track and playlist
```

Comment:
Emojis are not rendered within backticks, so we should either get rid of the code-formatted help or avoid specifying emoji. Here I've chosen the latter.

I would argue that a global help that shows all possible forms of every command is not that helpful and just becomes difficult to read.